lib/sdr_client/deposit/create_resource.rb

```diff
@@ -47,8 +47,7 @@ def accession?
       end
 
       def path
-        path = DRO_PATH
-        path += '?accession=true' if accession?
+        path = "#{DRO_PATH}?accession=#{accession?}"
         path
```

Comment:
I don't think we need this local variable anymore, right?

Reply:
true